mods/ts/sequences/misc.yaml

```diff
@@ -489,3 +489,83 @@ palet03:
 palet04:
 	idle:
 
+tracks01:
+	idle:
+		UseTilesetExtension: true
+		ZOffset: -1c511
```

Comment:
0.5 isn't:

![image](https://cloud.githubusercontent.com/assets/756669/9703238/62fe6b02-547d-11e5-9d75-6dd620a75a78.png)

1.5 is:

![image](https://cloud.githubusercontent.com/assets/756669/9703243/895998ee-547d-11e5-8f81-4d29c65f9ac1.png)